config/src/config/execution_config.rs

```diff
@@ -46,6 +48,8 @@ impl Default for ExecutionConfig {
             // Parallel execution by default.
             concurrency_level: 8,
             num_proof_reading_threads: 32,
+            paranoid_type_verification: true,
+            paranoid_hot_potato_verification: true,
```

Comment:
@sherry-x , we need to discuss the right values for this in our deployment configs before we should land this.

Reply:
We go now default to be on? Good with me.

Reply:
I think we will turn it on by default. We've tested it with all txns in testnet and no positives has been triggered